### DISCUSSION TITLE
直链后面的pw

### DISCUSSION THREAD

**Discussion by ghost** [AlistGo/alist#187] [Q&A] [answered]
能否设置直链取消后面的pw，这和我的很多设置冲突。
https://xxx.club/pdf/web/viewer.html?file=https://xxx.club/d/aliyun/Website/document/xxx.pdf?pw3dfb712837dee17781
譬如，无法正常打开了。因为有两个?号

**Comment by xhofe** [comment, marked as answer]
后台关闭check down link

**Reply by ghost** [reply]
谢谢
